### DISCUSSION TITLE
👋 Office Hours drop in sessions

### DISCUSSION THREAD

**Discussion by mcclowes** [codatio/codat-docs#323] [Announcements]
**We've paused these for now whilst we're working on some exciting other things, but we're open to bringing them back based on demand!**

Looking for advice on how to build with Codat?

Come along to an Office Hours session to discuss your project with our solutions expert, James Yorston (@jmyorston).

- Uncertain of the best approach to integrating customer spend with their accounting data?
- Perhaps you’re seeking guidance on processing accounting data to accelerate underwriting decisions without increasing your risk?
- Or maybe you need tips on effectively utilizing webhooks for managing data flow between applications?

Whatever you’re working on we’re looking forward to hearing about it!

---

#### Who's this for?

Whether you need a basic overview, or have specific questions about building integrations and use-cases, everyone is welcome!

#### What's the format?

Office Hours are 60 minute drop-in sessions where you can openly discuss your project with us and other Codat users.

**Comment by abcalderon3** [comment]
@mcclowes , Office Hours sessions sound great! Could you share a link with info on available times and how to join?

**Comment by mjp90** [comment]
@mcclowes I was able to grab some office hours via a link sent to me regarding a separate issue I raised, which was amazing! But how can I find a calendar to book office hours again? I've been banging my head on something with the way Codat handles web connector syncs for QB Desktop and could really use a quick call for some guidance 🙏

**Comment by mcclowes** [comment]
We've got three upcoming drop-in sessions, including one today(!):

- [May 9, 2023 4:00 PM (GMT+1) / 11:00 AM (ET)](https://codat-io.zoom.us/webinar/register/1016830363977/WN_qxrFBSvwRcKu_5j_VUG8qw?utm_campaign=Q22023_GROW_ALL_OfficeHours&utm_medium=email&_hsmi=2&_hsenc=p2ANqtz-_j1nRVHDEFtH3WLEQrFcojW-LKsnzqgFfeS0qWLwZRTNnoTZmPHcrUIIGUhmeBV8akbT_AACjIf0JZTa0Y6636OY3TZA&utm_content=2&utm_source=hs_email#/registration)
- [May 18, 2023 3:30 PM (GMT+1) / 10:30 AM (ET)](https://codat-io.zoom.us/webinar/register/5416830364724/WN_ekzZjPt6QX-RJy8_xL2j3w?utm_campaign=Q22023_GROW_ALL_OfficeHours&utm_medium=email&_hsmi=2&_hsenc=p2ANqtz--svxBWZSulBtgfedTaj2NU5V46gFCeNl2urZv0vcx5gY9l9Z3v4EArfG_akHHXFIqZ8cjWOURYNnm2j4Z9lUi8UPk7_Q&utm_content=2&utm_source=hs_email#/registration)
- [May 23, 2023 4:00 PM (GMT+1) / 11:00 AM (ET)](https://codat-io.zoom.us/webinar/register/9416830365422/WN_BTzWzi28QF2jg_IVrE1fIw?utm_campaign=Q22023_GROW_ALL_OfficeHours&utm_medium=email&_hsmi=2&_hsenc=p2ANqtz--wvU-E_gJxEwwwqfl2y-SHSmYwVeo0YdeV5yfc18qW1IkSW6Y4sL82y0vV5XRt982ZSlk_kAm0Dquz_65GuM6YxoZy4A&utm_content=2&utm_source=hs_email#/registration)

Hopefully we can answer those issues for you @mjp90 @abcalderon3 !

**Comment by mjp90** [comment]
Amazing!

Thanks for posting those and notifying me. Booked one of the times :)

I look forward to chatting.

Thanks,
Mike
